server_scripts/build_server_control_scripts.py

```diff
@@ -62,26 +62,25 @@ def getConfig(param, cfg, cfile):
 with open(os.path.join(serviceDir, 'start_service'), 'w') as ss:
     ss.write('export JAVA_HOME={}\n'.format(javaHome))
     ss.write('export PATH=$JAVA_HOME/bin:$PATH\n')
-    ss.write('export CLASSPATH=\n')
+    ss.write(('JARS={}/lib/jars\n').format(target))
     ss.write('if [ -z "$KB_DEPLOYMENT_CONFIG" ]\n')
     ss.write('then\n')
     ss.write('    export KB_DEPLOYMENT_CONFIG={}/deployment.cfg\n'
              .format(target))
     ss.write('fi\n')
-    ss.write(('{}/glassfish_administer_service.py --admin {} ' +
-        '--domain {} --domain-dir {}/glassfish_domain ' +
-        '--war {} --port {} --threads {} --Xms {} --Xmx {} ' +
-        '--properties KB_DEPLOYMENT_CONFIG=$KB_DEPLOYMENT_CONFIG\n')
-        .format(serviceDir, asadmin, serviceDomain, serviceDir,
-                os.path.join(serviceDir, war), port, threads, minmem, maxmem))
+    ss.write(('cd {}\n').format(serviceDir))
+    ss.write(('java -cp $JARS/jetty/jetty-start-7.0.0.jar:$JARS/jetty/jetty-all-7.0.0.jar:$JARS/servlet/servlet-api-2.5.jar ' +
+        '-Xmx{}m ' +
+        '-Djetty.port={} ' +
+        '-DKB_DEPLOYMENT_CONFIG=$KB_DEPLOYMENT_CONFIG ' +
+        'org.eclipse.jetty.start.Main jetty.xml\n')
+        .format(maxmem,port,serviceDir,serviceDir))
 
 with open(os.path.join(serviceDir, 'stop_service'), 'w') as ss:
     ss.write('export JAVA_HOME={}\n'.format(javaHome))
     ss.write('export PATH=$JAVA_HOME/bin:$PATH\n')
     ss.write('export CLASSPATH=\n')
-    ss.write(('{}/glassfish_administer_service.py --admin {} ' +
-        '--domain {} --domain-dir {}/glassfish_domain --port {}\n')
-        .format(serviceDir, asadmin, serviceDomain, serviceDir, port))
+    ss.write('killall java\n')
```

Comment:
What's the consequences if this doesn't get merged for (say) a couple months? Do we need to switch to Jetty right this second?

BTW, shock_service uses daemonize for this sort of thing which seems to work well.